lib/kitchen/patches/i18n.rb

```diff
@@ -4,6 +4,10 @@
 
 # rubocop:disable Style/Documentation
 module I18n
+  def self.group_by(character)
```

Comment:
`group_by` as a name makes sense in the context of indexes, but this function doesn't group anything, it just decides what character to return based on locale. Maybe a better name would be something like `transliterate_for_index` or `character_to_group`, or something I haven't thought of! lmk what you think :)

Reply:
Oh good point :+1: I didn't think about it. I like very much second proposition. It's more descriptive. I made a fix for that.